virttest/libvirt_xml/vm_xml.py

```diff
@@ -2038,25 +2167,59 @@ def marshal_to_caches(tag, new_treefile, index, libvirtxml):
         return newone
 
 
-class CellCacheXML(base.LibvirtXMLBase):
+class CellCacheXML(VMXML):
 
-    """Cell element of cache"""
```

Comment:
I don't see the necessity of this change, could you help provide more specific reason?

Reply:
It's best practice from me. @kylazhang , can you please help to explain it? Thanks.

Reply:
I think the element should all based on LibvirtXMLBase so keep the origin should be right

Reply:
After talked with kyla, "base.LibvirtXMLBase" is more better to be used. So, change to use "base.LibvirtXMLBase".  Thanks